sample/src/main/java/com/zachklipp/richtext/sample/ScreenPreview.kt

```diff
@@ -61,17 +59,9 @@ import androidx.compose.ui.unit.IntSize
     val previewDensityScale = constraints.maxWidth / screenSize.width.toFloat()
     val previewDensity = actualDensity * previewDensityScale
 
-    // Provide a fake host view, since the preview doesn't really belong to this host view.
-    val context = LocalContext.current
-    val previewView = remember {
-      val previewContext = context.applicationContext
-      View(previewContext)
```

Comment:
When I checked out the stack trace for this error, I found that the problem is that compose is looking for a ViewGroup instead of a basic View. So, my suggestion is to use `FrameLayout(previewContext)` and keep everything the same for now.

Reply:
Keeping _everything_ the same wouldn't work because the experimental ripple APIs have been removed since after beta-07.

Why would the hack still be needed? Did you try to run the sample app with your suggestion?

Reply:
Yeah I know those Ripple workarounds would have to be deleted because they do not exist anymore. I tested my approach and it works alright. My understanding for this particular hack was that slideshow sample would force the view to be landscape because it gets attached to the actual parent view. FrameLayout replacement keeps everything the same as before without causing a crash.

Reply:
Making it optional is a better option IMHO, for theee reasons:

1. Easier to embed it, say, for a slideshow at the top of a screen, no need to be aware of how to hack the thing.
2. When using the function, the autocompletion lets you know about the fact that it will make the app fullscreen while shown, and you can easily make the choice.
3. There's no "trojan-like" behavior where the composable has a visual effect beyond its boundaries that are not related to user input like a popup window would.

Reply:
Hey @LouisCAD , would you like to reopen a PR for specifically this change when you are available? Other changes are already incorporated into the main branch with 1.0.0 release.

Reply:
Hey Halil, yep, I'll do that this week.